AndrewTehan/hometask-6/scrapper/file_writer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Scrapper
+  # this class writes data into file
+  class FileWriter
+    REPORTS_PATH = 'reports/'
+    EXTENSION = '.csv'
+    HEADERS = ['Section', 'Title', 'Image URL'].freeze
+
+    attr_reader :report
+
+    def initialize(report)
+      @report = report
+    end
+
+    def call
+      file_name = Time.now.strftime('%d_%m_%Y_%H%M')
+      file_path = REPORTS_PATH + file_name + EXTENSION
```

Comment:
I'd move it to a separate method that will be responsible for the generation of a file's name